File: samples/java/src/app/Primitives.java
package app;

// UI settings
// Expression: x
//   Type: int
//     Layout: None
//     Save: Object Attributes

// Expression: x1
//   Type: Float
//     Layout: None
//     Save: Object Attributes

public class Primitives {
    public static void main(String[] args) {
        int x = 1;
        x = 2;
        x = 3;

        Float x1 = 1.1f;
        x1 = 2.2f;
        x1 = 3.3f;

        System.out.println("x: " + x + ", x1: " + x1);

        System.out.println("done");
    }
}

File: samples/ts/src/Primitives.ts
// UI settings
// Expression: x
//   Type: number
//     Layout: None
//     Save: Object Attributes

// Expression: x1
//   Type: number
//     Layout: None
//     Save: Object Attributes

class Primitives {
    public static runMain() {
        let x = 1;
        x = 2;
        x = 3;
        
        let x1 = 1.1;
        x1 = 2.2;
        x1 = 3.3;

        console.log(x, x1);
        console.log("done");
    }
}
Primitives.runMain();